make-it/cpplint/Makefile

```diff
@@ -1,4 +1,4 @@
-LINTARGS = --linelength=160 --filter=-build/header_guard,-readability/casting
+LINTARGS = --linelength=160 --filter=-build/header_guard,-readability/casting,-whitespace/braces
```

Comment:
In the next change I am going to remove this -readability/casting filter and replace all C-style casts with C++ style casts.